### ISSUE TITLE
NTR: Cholestanol high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8242]
New term request Cholestanol high in blood (segal_031009143354):

**Comment by pnrobinson** [comment]
@MickeySegal

is this part of human metabolism?
see https://en.wikipedia.org/wiki/Cholestenol

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK1409/ "The biochemical abnormalities that distinguish CTX from other conditions with xanthomas include high plasma and tissue cholestanol concentration, normal-to-low plasma cholesterol concentration, decreased chenodeoxycholic acid (CDCA), increased concentration of bile alcohols and their glyconjugates, and increased concentrations of cholestanol and apolipoprotein B in cerebrospinal fluid."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000527
- label: Elevated circulating cholestanol concentration
- parents: [Abnormal circulating cholesterol concentration (HP:0003107)](https://hpo.jax.org/app/browse/term/HP:0003107)
- pmid: 
- def: The concentration of cholestanol in the blood circulation is above the upper limit of normal. Cholestanol is the 5-alpha-dihydro derivative of cholesterol.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8242
- synonyms:
